### ISSUE TITLE
powerline

### ISSUE THREAD

**Issue by atrauzzi** [powerline/powerline#850]
I'm getting the following message when trying to configure Powerline with Bash on Ubuntu 13.10:

```
bash: /home/ME/.local/lib/python2.7/site-packages/powerline/bindings/bash/../../../scripts/powerline: No such file or directory
```

**Comment by tomleo** [comment]
I had the same issue this is how I was able to get things working.

In `~/.bashrc`

```sh
if [ -d "$HOME/.local/bin" ]; then
    PATH="$HOME/.local/bin:$PATH"
fi
export POWERLINE_COMMAND=powerline
```

Hope that might help.


System Info `cat /etc/*release`
```
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=14.10
DISTRIB_CODENAME=utopic
DISTRIB_DESCRIPTION="Ubuntu 14.10"
NAME="Ubuntu"
VERSION="14.10 (Utopic Unicorn)"
```

**Comment by Goddard** [comment]
Having this same error in Ubuntu 15.10

Although none of the solutions above fixed this issue.

which powerline returns
/home/goddard/.local/bin/powerline

I tried going to this directory, but it doesn't exist
/home/goddard/.local/lib/python2.7/site-packages/powerline/bindings/bash/../../../scripts

**Comment by jakies** [comment]
Here's what I did to get things working on Ubuntu 15, using `pip` installation.

```sh
jake$ which powerline-config
~/.local/bin/powerline-config
```

Then, in my `~/.bashrc`
```sh
export PATH="$HOME/.local/bin:$PATH"
export POWERLINE_COMMAND=powerline
export POWERLINE_CONFIG_COMMAND=powerline-config
powerline-daemon -q
POWERLINE_BASH_CONTINUATION=1
POWERLINE_BASH_SELECT=1
. ~/.local/lib/python2.7/site-packages/powerline/bindings/bash/powerline.sh
```

**Comment by vrkansagara** [comment]
@jakies  it's work for me. on 

`Distributor ID:	Ubuntu`
`Description:	Ubuntu 14.04.3 LTS`
`Release:	14.04`
`Codename:	trusty`

**Comment by justn6** [comment]
Was having same issue, works for me

**Comment by saintplay** [comment]
Thanks
```bash
if [ -d "$HOME/.local/bin" ]; then
    PATH="$HOME/.local/bin:$PATH"
fi
export POWERLINE_COMMAND=powerline
```

On my .bashrc works on my Bash On Windows (Ubuntu)

**Comment by liuyanjie** [comment]
It doesn't work when install by pip

```sh
pip install powerline-status
```

But work after

```sh
sudo apt install powerline
```

Fixed on Ubuntu 16.04.1 LTS

**Comment by vrkansagara** [comment]
hum!
